components/Chips/tests/unit/ChipAccessibilityTests.m

```diff
@@ -0,0 +1,80 @@
+//
+//  ChipAccessibilityTests.m
+//  MaterialComponentsUnitTests
+//
+//  Created by Galia Kaufman on 6/18/18.
+//
+
+#import <XCTest/XCTest.h>
+#import "MaterialChips.h"
+
+@interface ChipAccessibilityTests : XCTestCase
+
+@end
+
+@implementation ChipAccessibilityTests {
+  MDCChipView *chip;
+}
+
+- (void)setUp {
+  [super setUp];
+  chip = [[MDCChipView alloc] init];
+}
+
+- (void)tearDown {
+    // Put teardown code here. This method is called after the invocation of each test method in the class.
+    [super tearDown];
+}
+
+- (void)testAccessibilityLabel {
+  NSString *chipLabel = @"Test1";
+  [chip.titleLabel setText:chipLabel];
+  XCTAssertTrue(chip.isAccessibilityElement, @"Chip view should be the accessibility element");
```

Comment:
Super-nit: You shouldn't need explicit comments if the test assertion will be sufficiently descriptive.  Hopefully it will save you time from writing all the messages. :)